docs/en/observability/logs-stream.asciidoc

```diff
@@ -314,17 +334,20 @@ After creating your ingest pipeline, create an index template to point your log
 ----
 PUT _index_template/logs-example-default-template
 {
-  "index_patterns": [ "logs-example-default" ],
+  "index_patterns": [ "logs-example-*" ],
   "data_stream": { },
   "priority": 500,
   "template": {
     "settings": {
       "index.default_pipeline":"logs-example-default"
     }
-  }
+  },
+  "composed_of": ["ecs@dynamic_templates"],
```

Comment:
@eyalkoren @felixbarny I suggest we recommend users to add the ecs template by default?

Reply:
I'll add it to the template. It means we also have an opportunity here to quickly explain why these are all here.